Doc/whatsnew/3.11.rst

```diff
@@ -262,15 +272,20 @@ See :pep:`655` for more details.
 (Contributed by David Foster and Jelle Zijlstra in :issue:`47087`. PEP
 written by David Foster.)
 
+
+.. _whatsnew311-pep673:
+
 PEP 673: ``Self`` type
 ----------------------
 
 The new :data:`~typing.Self` annotation provides a simple and intuitive
 way to annotate methods that return an instance of their class. This
-behaves the same as the :data:`~typing.TypeVar`-based approach specified
-in :pep:`484` but is more concise and easier to follow.
+behaves the same as the :data:`~typing.TypeVar`-based approach
```

Comment:
Should this be
```suggestion
behaves the same as the :class:`~typing.TypeVar`-based approach
```
as well?

Reply:
Indeed it should—I missed a spot here. Thanks!